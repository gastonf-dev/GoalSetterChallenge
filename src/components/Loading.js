/**
 * @flow stric-local
 * @format
 */

import React from 'react';
import {StyleSheet, View, ActivityIndicator} from 'react-native';
import COLORS from '../styles/Colors';

const Loading = () => {
  return (
    <View style={styles.loadingContainer}>
      <ActivityIndicator color={COLORS.lightblue} size="large" />
    </View>
  );
};

export default Loading;

const styles = StyleSheet.create({
  loadingContainer: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
});
