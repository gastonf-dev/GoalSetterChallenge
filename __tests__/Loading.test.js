import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../src/components/Loading';

describe('<Loading />', () => {
  const wrapper = renderer.create(<Loading />);

  test('render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
