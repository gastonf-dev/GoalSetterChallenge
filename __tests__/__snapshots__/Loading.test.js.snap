// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#17BFDD"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
